src/main/java/mate/academy/dao/impl/TicketDaoImpl.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.dao.impl;
+
+import mate.academy.dao.TicketDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Ticket;
+import mate.academy.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class TicketDaoImpl implements TicketDao {
+    @Override
+    public Ticket add(Ticket ticket) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(ticket);
+            transaction.commit();
+            return ticket;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't insert ticket to DB" + ticket, e);
```

Comment:
```suggestion
            throw new DataProcessingException("Can't insert ticket to DB: " + ticket, e);
```
it's better to add a whitespace before concatenation